### PR TITLE
Add transcripts

### LINKED ISSUE #186: Add transcript section to workshop pages
On individual workshop pages ([for example](https://learn.wordpress.org/workshop/user-management/)), it would be helpful to be able to display the video transcript in its own section for viewers to follow along or print for reference.

Would it be possible to add a section (either in the sidebar or a toggle) to display the transcript in full when activated?

In thinking through how to handle transcripts, it will be helpful to also consider cases in which the video is captioned or subtitled in multiple languages.

Just in case a visual is helpful, this is a very rough idea of where we could consider placement: 

transcript UI

### PR DESCRIPTION
**Pull request by iandunn**
Closes #186 

This is a simple approach that just adjusts the PHP-defined template, since #195 didn't work out. The basics are working, but I'm planning to make some other block/template adjustments that will affect this PR, and it'll be better to do those first. I'll open a PR for those later today or next week, and circle back to this after.

I started with @gwwar's idea about putting the transcript below the fold. It seems like dealing with w/ tabs inside the editor would be too complicated, but I might take a second look at that once I circle back.

This assumes that #198 will be merged first.

### CODE REVIEW

**Comment by iandunn** [comment]
I've got it working with the transcript at the bottom of the regular content, but I'm wondering if folks would prefer to have it beside the video, so they can scroll through it and watch the video at the same time?

What do you think, @melchoyce, @varlese , @coreymckrill ?

### Below Other Content

Screen Shot 2021-04-26 at 09 03 54-fullpage

----

### Next to video (very rough sketch)

There'd be a show/hide button, and it'd be hidden by default.

Screen Shot 2021-04-26 at 09 12 50-fullpage

**Comment by melchoyce** [comment]
Is the transcript in addition to video captions? If so, we won't need to include it alongside the video, underneath is fine IMO 👍

**Comment by coreymckrill** [comment]
Agree with Mel, below is fine. And since the videos are hosted at wp.tv, I think they do have (at least the possibility) of transcriptions.

File: wp-content/plugins/wporg-learn/views/block-workshop-details.php [hunk 1]
```diff
@@ -17,6 +23,14 @@
 					<span><?php echo esc_html( $value ); ?></span>
 				</li>
 			<?php endforeach; ?>
+
+			<?php if ( $has_transcript ) : ?>
+				<li>
+					<a class="components-button is-secondary is-small" href="#transcript">
```
Comment (coreymckrill) [inline wp-content/plugins/wporg-learn/views/block-workshop-details.php]: This button looks kind of out-of-place in the Workshop Details list. What if it matched the other detail rows more, something like this?

transcript

**Review by coreymckrill — APPROVED** [review]
👍 Everything works as expected. One non-blocking quibble with the "Jump to Transcript" button...

File: wp-content/plugins/wporg-learn/views/block-workshop-details.php (hunk 1 above)
Comment (iandunn) [reply wp-content/plugins/wporg-learn/views/block-workshop-details.php]: I like that, done in 708cf87ca32cb6d665293d3da990984a7d9fd870

**Pull request merged**